website/assets/styles/pages/configuration-generator.less

```
@@ -0,0 +1,180 @@
+    padding: 48px;
+  }
+
+  [purpose='page-content'] {
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 1104px;
+    color: #515774;
+  }
+  .ace-editor-container {
+
+  }
```

Stylelint (17.14.0)

[error] Empty block (block-no-empty)

(block-no-empty)